### PR TITLE
Generate changelogs on release

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-0

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

This is a part of the community hackday initiative. Fill release notes from pmm-doc automatically once new tag is generated

### CODE REVIEW

File: .github/workflows/release-doc.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+          cd doc/docs/release-notes
+          version="${release_tag/refs\/tags\/v//}"
+          CHANGELOG=`cat ./$version.md`
+          echo "CHANGELOG<<EOF" >> $GITHUB_ENV
+          echo "$CHANGELOG" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
+        env:
+          release_tag: ${{ github.ref }}
+      - name: Create Release
+        uses: softprops/action-gh-release@v0.1.14
```
Comment (artemgavrilov) [inline .github/workflows/release-doc.yml:24]: Please ask somebody with proper permissions to clone this action to percona namespace. We don't allow external action in our workflows.

File: .github/workflows/release-doc.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Build Changelog
+        id: pmm_release
+        run: |
+          git clone --depth 1 https://github.com/percona/pmm-doc doc
+          cd doc/docs/release-notes
+          version="${release_tag/refs\/tags\/v//}"
+          CHANGELOG=`cat ./$version.md`
+          echo "CHANGELOG<<EOF" >> $GITHUB_ENV
+          echo "$CHANGELOG" >> $GITHUB_ENV
+          echo "EOF" >> $GITHUB_ENV
```
Comment (artemgavrilov) [inline .github/workflows/release-doc.yml:20]: Not sure, but probably job output will fit better: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-output-parameter

Comment (artemgavrilov) [reply .github/workflows/release-doc.yml:20]: It seems there is solution even better: https://github.com/softprops/action-gh-release#-external-release-notes

Comment (gen1us2k) [reply .github/workflows/release-doc.yml:20]: ::set-output does not work with multiline strings

File: .github/workflows/release-doc.yml (hunk 1 above)
Comment (BupycHuk) [reply .github/workflows/release-doc.yml:22]: @atymchuk can we clone this release?

Comment (atymchuk) [reply .github/workflows/release-doc.yml:22]: @BupycHuk Done, you can try now

File: .github/workflows/release-doc.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build Changelog
+        id: pmm_release
+        shell: bash
+        run: |
+          git clone --depth 1 https://github.com/percona/pmm-doc doc
+          cd doc/docs/release-notes
+          version="${release_tag/refs\/tags\/v//}"
+          cat ./$version.md > ${{ github.workspace }}-CHANGELOG.txt
```
Comment (artemgavrilov) [inline .github/workflows/release-doc.yml:18]: It looks like we can upload `$version.md` directly.

Comment (gen1us2k) [reply .github/workflows/release-doc.yml]: Nikita suggested using raw. I simplified it and do not use git clone.

File: .github/workflows/release-doc.yml (hunk 1 above)
Comment (artemgavrilov) [reply .github/workflows/release-doc.yml:20]: And what it the idea?

Comment (BupycHuk) [reply .github/workflows/release-doc.yml:20]: Adding softprops/action-gh-release to allowed github actions list :)

File: .github/workflows/release-doc.yml (hunk 3 above)
Comment (artemgavrilov) [reply .github/workflows/release-doc.yml]: I mean what is the reason to rename file to `*-CHANGELOG.txt` ? It's still Markdown.

File: .github/workflows/release-doc.yml (hunk 1 above)
Comment (artemgavrilov) [reply .github/workflows/release-doc.yml:20]: But why? That creates security breach, no?

File: .github/workflows/release-doc.yml (hunk 3 above)
Comment (gen1us2k) [reply .github/workflows/release-doc.yml]: It's still markdown, but ${{ github.workspace }} is a directory. To make it predictable between steps I add `-CHANGELOG.txt` and upload it using the next pipeline step.

File: .github/workflows/release-doc.yml (hunk 1 above)
Comment (artemgavrilov) [reply .github/workflows/release-doc.yml:20]: Discussed with Alex. We will use pinned versions + constraints in repo.

**Review by artemgavrilov — APPROVED** [review]

**Review by BupycHuk — APPROVED** [review]

**Pull request merged**